lib/query/country_divisions.rb

```diff
@@ -39,5 +39,9 @@ def sparql
     def division_results
       @res ||= Sparql.new(sparql).results
     end
+
+    def legislatures(place_id)
+      division_results.select { |r| r[:item] == place_id }.map { |i| i[:legislature] }.uniq
```

Comment:
Is the `.uniq` at the end of this needed? I can't quite figure out when/why it is, and the tests pass without it, so if there _is_ a scenario where it's important, I think we need a new test.

Reply:
Added back in with a test in 0b122ee282030ce6c353e9c21f2f956e181e83e0. The example I found was Jerusalem being listed once with 9 of the same legislatures.